src/transformers/models/opt/convert_opt_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -37,8 +37,6 @@ def load_checkpoint(checkpoint_path):
     # pop unnecessary weights
     keys_to_delete = [
         "decoder.version",
-        "decoder.layer_norm.weight",
-        "decoder.layer_norm.bias",
         "decoder.output_projection.weight",
```

Comment:
This was ignored because in opt-125 and opt-350 the models use the input embedding layer to project it out to the vocabulary, see https://github.com/facebookresearch/metaseq/blob/e0c4f6b0e4c523906ad8d561f727e3f2ac3a8e73/metaseq/models/transformer.py#L486 
Since models can have different configs, I think it is better to double check that with the authors?

Reply:
Yep I sort of assumed that all models had that value to True. But indeed, didn't check besides 125M and 350M.